packages/common/src/networks/data.json

```diff
@@ -105,16 +105,16 @@
       "chainId": 137,
       "address": "0x0000000000000000000000000000000000000000",
       "decimals": 18,
-      "symbol": "MATIC",
-      "name": "Matic Token",
+      "symbol": "POL",
+      "name": "Polygon Ecosystem Token",
       "logoUri": "https://cdn.furucombo.app/assets/img/token/MATIC.png"
```

Comment:
should change the` logoUri` as well?

Reply:
updated and uploaded `POL` `WPOL` image.